models/articles-models.js

```diff
@@ -1,5 +1,31 @@
 const db = require("../db/connection");
 
+exports.selectArticles = (sort_by = "created_at") => {
+  const ValidSortBys = ["created_at", "votes"];
```

Comment:
could add a few more to validSortBys (remember to use correct camelcasign as well.  will help to sort by other properties on your front end project.